src/theme/index.js

```diff
@@ -50,6 +52,10 @@ const theme = createTheme({
       secondary: "#FFFFFF",
       hint: "#1C2031",
     },
+    chart: {
+      text: "#707070",
+      category: CHART_PRIMARY_COLOR_SCHEME,
+    },
```

Comment:
Why not replicate MUI approach i.e. `chart: { primary: ..., secondary: ..., text: { primary: ..., secondary: ..} }`?